src/app/deleted/deleted.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { UsersService } from '../users.service';
+
+@Component({
+  selector: 'app-deleted',
+  templateUrl: './deleted.component.html',
+  styleUrls: ['./deleted.component.css']
+})
+export class DeletedComponent {
+
+  deletedUsers = [];
+
+  constructor(private users: UsersService) {}
+
+  ngDoCheck(){
+    this.deletedUsers = this.users.getInActiveUsers();
+  }
+
+  activate(id: string) {
+    this.users.changeStatus(id);
```

Comment:
Once you activated or de-activated a user, are they reflecting the changes immediately ? I mean if you activate a user, that should be removed from active-users list and should be shown in deleted-users list. Is that case being handled ?

Reply:
yes they are. for that reason i am use ngDoCheck